packages/apps/contracts/PreimageApp.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity 0.5.8;
+pragma experimental "ABIEncoderV2";
+
+import "@counterfactual/contracts/contracts/interfaces/CounterfactualApp.sol";
+
+
+contract PreimageApp {
+
+  struct AppState {
+    bytes32 image;
+    bool preimageRevealed;
+  }
+
+  struct Action {
+    bytes preimage;
+  }
+
+  function isStateTerminal(bytes memory encodedState)
+    public
+    pure
+    returns (bool)
+  {
+    AppState memory state = abi.decode(encodedState, (AppState));
+    return state.preimageRevealed;
+  }
+
+  function getTurnTaker(
+    bytes memory /* encodedState */, address[] memory signingKeys
```

Comment:
Does omitting the variable instantiation do anything?

Reply:
It's an unused parameter, so this silences the warning